modules/programs/ssh.nix

```diff
@@ -6,11 +6,17 @@ let
 
   cfg = config.programs.ssh;
 
+  addressPort = entry:
+    let
+      isPath = x: builtins.substring 0 1 (toString x) == "/";
+      addressIsPath = isPath entry.address;
```

Comment:
Similarly, I'd like to assert that there was no port set when the address is a path.